src/lib.rs

```diff
@@ -92,14 +85,49 @@ pub enum TeePubKey {
     },
 }
 
+/// Data generated during the attestation process between client and server. Relevant only to the
+/// client-server pairing.
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct RuntimeData {
+    /// Nonce string generated by server.
+    pub nonce: String,
+
+    /// TEE public key generated by client.
+    #[serde(rename = "tee-pubkey")]
+    pub tee_pubkey: TeePubKey,
+}
+
+/// Combined evidence of all TEE devices found within a client.
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(rename_all = "kebab-case")]
```

Comment:
any reason to add this macro here?

Reply:
The `Attestation` fields were all kebab-case, so this enforces it to stay uniform.

Reply:
Ok. What I worried is [here](https://github.com/confidential-containers/kbs-types/pull/71/files#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759R105). Originally we do not mark it as `kebab-case` thus the serialization would be `_`. Now if we explicitly claims as `kebab-case` it will be `-`. Thus causing compatibility issue.

My idea is to just remove the macro and keep it as what original [definition](https://docs.rs/kbs-types/0.12.0/src/kbs_types/lib.rs.html#87-92) is.

Reply:
No problem, I deleted the kebab case from this group.